quicklogic/common/cmake/quicklogic_board.cmake

```diff
@@ -132,7 +132,7 @@ function(ADD_QUICKLOGIC_BOARD)
     add_custom_command(
       OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/${CLKMAP_CSV}
       COMMAND
-        ${CMAKE_COMMAND} -E create_symlink 
+        ${CMAKE_COMMAND} -E copy
```

Comment:
why do we need to copy this?

Reply:
Symlinking works when running everything from the repo using CMake but creates issues with installation. Basically CMake installs the symlink itself rather than the real file. There is no option in the CMake `install` command like eg. "resolve symlinks". I couldn't solve that in any way in a CMake script as at the  generation stage the link isn't present yet so it cannot be resolved and I cannot identify the real path of a file to pass it to the `install` command.

And more generally, the file is symlinked/copied to maintain consistency of SymbiFlow across all architectures. I could make the flow use the original file directly but that would require more changes to other CMake scripts.